### DISCUSSION TITLE
Surface pro 8 doesnt boot into linux

### DISCUSSION THREAD

**Discussion by Starfirestorm1** [linux-surface/linux-surface#1328] [Q&A]
Greetings, one strange thing happened.
After I installed the kernel boot stack at "launching EFI/boot/bootx64.efi". Ubuntu version 22.04, tried also version 23 made with rufus
![IMG_20231221_225038](https://github.com/linux-surface/linux-surface/assets/147154636/4bae2d66-5eeb-4855-9547-9badedb2bd0a)

**Comment by vistis** [comment]
Hi! I hope it's not too late. I think the culprit might be `Rufus`. Rufus isn't recommended to create Linux bootable flash drives. It specializes at creating Windows bootable flash drives, from the picture, it looks like it's using NTFS filesystem, which might be why it's not working properly. Try other tools like `Balena Etcher`. Also, make sure you turn off secure boot or allow 3rd party before trying to boot into your flash drive.
